### ISSUE TITLE
`hide: - path` in front matter does not hide the navigation path

### ISSUE THREAD

**Issue by Valerie-ts** [squidfunk/mkdocs-material#6705]
### Context

I use breadcrumbs in my documentation, but want to hide them on the index page of my folders.
The option to hide bradcrumbs on single pages was introduced as part of 9.0.5-insiders-4.28.1, see [#4875](https://github.com/squidfunk/mkdocs-material/issues/4875).

### Bug description

The front matter of a file contains `hide: - path`, but the navigation path is still shown on the page.
![image](https://github.com/squidfunk/mkdocs-material/assets/79153960/d852e71c-77a3-4e22-af1f-9a4ab33b3d42)

### Related links

- [#4875](https://github.com/squidfunk/mkdocs-material/issues/4875)
- [Documentation: Setting up Navigation - Navigation path](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-path)

### Reproduction

[9.5.6+insiders.4.52.0-hide-breadcrumbs.zip](https://github.com/squidfunk/mkdocs-material/files/14076210/9.5.6%2Binsiders.4.52.0-hide-breadcrumbs.zip)

### Steps to reproduce

1. Add the following configuration to the mkdocs.yml file:
```
theme:
  name: material
  features:
     - navigation.path
```
2. Create a folder "breadcrumbs" in the /docs directory.
3. Create a new index.md file in the folder.
4. Add the following front matter:
```
---
hide:
  - path
---
```
5. Serve the project.

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by alexvoss** [comment]
The `hidden` attribute on the `nav` element for the breadcrumbs is set correctly but the browser (tried Firefox and Safari) still displays the breadcrumbs. It seems that `hidden` is overridden by CSS styling as [it merely triggers browser built-in styling](https://css-tricks.com/the-hidden-attribute-is-visibly-weak/)? The `.md-path` class applied to `nav` has `display="block"`. I'll go out on a limb here and label this a bug.

**Comment by squidfunk** [comment]
Thanks for reporting and providing the reproduction! Fixed in [`1dfc91e`](https://github.com/squidfunk/mkdocs-material-insiders/commit/172f76b5e1a4f5b220156f387fec5d5d82b306c2).

**Comment by squidfunk** [comment]
Released as part of [9.5.6+insiders-4.52.1](https://squidfunk.github.io/mkdocs-material/insiders/changelog/#4.52.1).
